Kconfig.nrf

```diff
@@ -11,9 +11,23 @@ menu "Nordic nRF Connect"
 config BOARD_THINGY91_NRF9160NS
 	bool
 
-# nRF Connect SDK needs a larger default stacks for running certain tests.
+# Handle default to ensure that BT only will disable ENTROPY_CC3XX,
+# while multi protocol does not.
+config ENTROPY_CC3XX
+	default y if ZIGBEE || OPENTHREAD_NRF_SECURITY
+	default n if BT
+
+config ZIGBEE
+	select ENTROPY_CC3XX
+
+config OPENTHREAD_NRF_SECURITY
+	select ENTROPY_CC3XX
```

Comment:
Isn't the `default` statement in line `17` enough? Is the `select` needed? I think Kconfig gives more meaningful errors in case of misconfiguration when `depends` is used instead.

Reply:
This has been removed